common/helpers

```diff
@@ -825,7 +825,13 @@ print_msgs ()
 
 get_local_upstream_dns ()
 {
-    local server="`systemd-resolve --status 2>/dev/null| sed -r 's/\s+DNS Servers:\s+([[:digit:]]+)/\1/g;t;d'| head -n 1`"
+    # Use old resolve command if present
+    if [ -f /usr/bin/systemd-resolve ]; then
+        local resolver="systemd-resolve --status"
+    else
+        local resolver="resolvectl"
+    fi
+    local server="`$resolver 2>/dev/null| sed -r 's/\s+DNS Servers:\s+([[:digit:]]+)/\1/g;t;d'| head -n 1`"
     (($?==0)) && echo $server && return 0
     echo "10.198.200.1"  # stsstack upstream dns
```

Comment:
```suggestion
    # Use old systemd-resolve command if present
    if command -v systemd-resolve > /dev/null; then
        local server=$(systemd-resolve --status 2> /dev/null | sed --regexp-extended 's/\s*DNS Servers:\s+([[:digit:]]+)/\1/g;t;d'| head -n 1)
    else
        local server=$(resolvectl 2> /dev/null | sed --regexp-extended 's/\s*Current DNS Server:\s+([[:digit:]]+)/\1/g;t;d')
    fi
    (($? == 0)) && echo $server && return 0
    echo "10.198.200.1"  # stsstack upstream dns
```

Reply:
After more careful testing I realized that the output format is slightly different between the two commands.